examples/openshift-lightspeed-tls.yaml

```diff
@@ -139,6 +139,9 @@ data:
           - name: gpt-4-1106-preview
           - name: gpt-3.5-turbo
     ols_config:
+      reference_content:
+        product_docs_index_path: "./vector-db/ocp-product-docs"
+        product_docs_index_id: product
```

Comment:
embeddings_model_path: ./embeddings_model?

Reply:
good point, added

Reply:
sorry, i thought i pushed this change but apparently not.  adding it here: https://github.com/openshift/lightspeed-service/pull/514